components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.html

```diff
@@ -14,27 +14,87 @@
-        <div id="left-align">
+        <div id="modal-content-container">
           <form [formGroup]="expressionForm">
+
+              <chef-form-field id="create-type">
+                <label>
+                  <span class="label">Type<span aria-hidden="true">*</span></span>
+                  <select id="expression-type-dropdown"
+                          ngDefaultControl 
+                          (change)="displayExpressionOutput()"
+                          formControlName="type">
+                    <option value='user'>User</option>
+                    <option value='team'>Team</option>
+                    <option value='token'>Token</option>
+                    <option value='*'>All members (*)</option>
+                  </select>
+                </label>
+                <chef-error *ngIf="expressionForm.get('type').hasError('required') && expressionForm.get('type').touched">
+                  Type is a required field.
+                </chef-error>
+              </chef-form-field>
+
+              <chef-form-field 
+                id="create-identity" 
+                *ngIf="showInputs('identity')"
+                [@dropInAnimation]="{ value: triggerValue, params: { height: 74.6875 }}" >
+                <label>
+                  <span class="label">Identity Provider<span aria-hidden="true">*</span></span>
+                  <select id="expression-identity-dropdown" 
+                          ngDefaultControl 
+                          (change)="displayExpressionOutput()"
+                          formControlName="identity">
+                    <option value="ldap">LDAP</option>
+                    <option value="local">local</option>
+                    <option value="saml">SAML</option>
+                    <option value="*">All {{ allIdentity | titlecase }}s (*)</option>
+                  </select>
+                </label>
+                <chef-error *ngIf="expressionForm.get('identity').hasError('required') && expressionForm.get('identity').touched">
+                  Identity Provider is a required field.
+                </chef-error>
+              </chef-form-field>
+
+              <chef-form-field 
+                id="create-name" 
+                *ngIf="showInputs('name')"
+                [@dropInAnimation]="{ value: triggerValue, params: { height: 119.427 }}">
+                <label>
+                  <span class="label">{{ nameOrId }}<span aria-hidden="true">*</span></span>
+                  <chef-input id="expression-name-dropdown" 
+                              ngDefaultControl 
+                              (keyup)="displayExpressionOutput()"
+                              formControlName="name"></chef-input>
+                </label>
+                <p class="note">Wildcards (*) are allowed.</p>
+
+                <chef-error *ngIf="expressionForm.get('name').hasError('required') && expressionForm.get('name').touched">
+                  Name is a required field.
```

Comment:
1. This is not working--an empty field does not display the error pop-up. (Works ok for for Type and Identity Provider.
2. Need an additional `<chef-error>` to check for `hasError('pattern')` with the text `Name must be a user name or a wildcard but not both`

@susanev Need a check on that messaging: Normally we would say "Only lowercase letters, numbers, hyphens are allowed", but this needs to allow any LDAP or SAML user name, too, but LDAP at least allows all sorts of punctuation characters, too. So that's why I think just saying "a user name" conveys sufficient information.